ocaml/xapi/extauth_plugin_ADpbis.ml

```diff
@@ -563,9 +556,18 @@ struct
     in
     begin
       debug "Testing if external authentication server is accepting requests...";
+      let full_username = get_full_subject_name username in
+      begin try
+          ignore(pbis_common "/opt/pbis/bin/ad-cache" ["--delete-user"; "--name"; full_username])
+        with _ -> ()
```

Comment:
I believe in case of an exception, this should be logged. Should this not also mean that the function should return false?

Reply:
I agree about logging the exception.
What happens if the user we ask to delete is not in the cache, would pbis fail? I couldn't find a manpage for `ad-cache` that would document its behaviour.

Reply:
Sure. I will log the exception to make it more clear.
If the user does not exist in cache, pbis will fail. This deletion is just to make sure it is not in cache. This forces `test 0` to retrieve the user from remote server, rather than cache.